javascript-data-model/order-history.js

```diff
@@ -0,0 +1,67 @@
+var orderHistory = [
+  {
+    orderDate: 'August 4, 2020',
+    total: '$34.00',
+    shipTo: 'JS Masher',
+    orderNumber: '114-3941689-8772232',
+    deliveryDate: 'Aug 8, 2020',
+    items: [
+      {
+        itemName: {
+          title: 'Javascript for impatient programmers',
+          author: 'Rauschmayer, Dr. Axel'
+        },
```

Comment:
This is an object for books, but a string for other items. Instead, you should always save the item name as a string for consistency, and maybe add a property named `data` that will hold an object with specific data for that item, such as author.